### ISSUE TITLE
Error when pie chart group becomes really small

### ISSUE THREAD

**Issue by shusson** [mapd/mapd-charting#67]
## :beetle:
  - [x] Operating System and version:
macOS 10.12.5
  - [x ] Browser and version:
Chrome 59.0.3071.115
  - [x] Mapd versions
mapdc tag v1.1.1
mapd-crossfilter tag v1.1.0
mapd-connector tag v2.0.0
  - [x] Steps to reproduce (including dashboard link):
    - go to https://github-investigation.firebaseapp.com/
    - click on the user `greenkeeperio-bot`
  - [x] Description of issue:
The `merged` pie chart goes into the loading state indefinitely 
Console shows an error like: 
```
": error"
TypeError: t.replace is not a function
```

We are seeing this on a different project as well.

**Comment by shusson** [comment]
Has anyone been able to reproduce this?

**Comment by duhoang** [comment]
Yes, thank you for bringing it to our attention. We now have a ticket internally to fix this bug. Thanks!

**Comment by shusson** [comment]
hi @duhoang, any updates on this issue?

**Comment by duhoang** [comment]
Hi @shusson,

Thank you for following up on this. This ticket got lost in the shuffle, but we are planning to work on it this sprint, so look for a fix in the next couple of releases.
